model/src/main/proto/exploration.proto

```diff
@@ -246,17 +246,9 @@ message UserAnswer {
     string plain_answer = 3;
     // A list of HTML textual representation of the answer.
     ListOfSetsOfHtmlStrings list_of_html_answers = 4;
-    // Contains a plain answer with an accessible version of the same answer.
-    AccessibleAnswer accessible_answer = 5;
   }
-}
-
-message AccessibleAnswer {
-  // A plaintext representation of the answer.
-  string plain_answer = 1;
-
   // A accessible version of the corresponding plain answer.
```

Comment:
Please define how this is used, and how the default value is treated.

Reply:
Didn't get about the default value?

Reply:
What I mean is: the documentation should indicate what happens if this value is uninitialized. In proto, that means the string is empty & we need to explicitly handle that case (e.g. use the plain answer as the accessible answer, instead). The documentation should make it clear so that users of the proto know how to initialize it properly.

Reply:
Understood would make the changes